doc/authentication-backends/JWT-authentication-module.md

```diff
@@ -0,0 +1,31 @@
+## Overview
+
+JWT authentication backend can verify JSON Web Tokens provided by the clients.
+A wide range of signature algorithms is supported, including those using public key cryptography.
+Not only the signature is checked, but also the validity of `exp` (expired token is rejected), `iat` (token must be issued in the past) and `nbf` (token might be invalid *yet*) parameters.
+
+Requires SASL PLAIN method.
+
+## Configuration options
+
+* **jwt_secret_source**
+    * **Description:** A path to a file or environment variable, which contents will be used as a JWT secret.
```

Comment:
**Description:** A path to a file or an environment variable, which contents will be used as a JWT secret.